web/frontend/src/components/camera/camera-manager.ts

```diff
@@ -40,6 +42,7 @@ export class CameraManager {
         return;
       }
       this.videoStream = eventStream;
+      this.onOpen?.();
```

Comment:
Fixes an issue with camera live feeds. Since we reset the video stream on each `track` event, previously there was a chance that the `srcObject` on our video element would be come detached from `videoStream` because it isn't reactive. There are much better long term solutions but at least by firing and listening to this event we can ensure that our video element is always attached to the correct stream object.